### PR TITLE
Upgrade dependencies

### PR DESCRIPTION
**Pull request by kyam-mayden**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -29,10 +29,10 @@
       }
     },
     "require-dev": {
-        "phpunit/phpunit": "^5"
+        "phpunit/phpunit": "^8.0"
```
Comment (imjoehaines) [inline composer.json]: Could this be v9?

```suggestion
        "phpunit/phpunit": "^9.0"
```

File: composer.json [hunk 2]
```diff
@@ -29,10 +29,10 @@
...
     },
     "require": {
-        "imjoehaines/flowder": "^1.0",
-        "codeception/codeception": "^2.1"
+        "imjoehaines/flowder": "^2.0",
```
Comment (imjoehaines) [inline composer.json:35]: This only allows PHP 7.3 or 7.4; do you need it to run on PHP 8 too?

File: composer.json (hunk 1 above)
Comment (imjoehaines) [reply composer.json]: It'd be good to add the `.phpunit.result.cache` file to gitignore as well

Comment (kyam-mayden) [reply composer.json]: Could be - Wasn't sure if you wanted to support V8 as it's not EOL yet

File: composer.json (hunk 2 above)
Comment (kyam-mayden) [reply composer.json:35]: 7.4 should be enough for us, for now

File: composer.json (hunk 1 above)
Comment (kyam-mayden) [reply composer.json]: gitignore updated - 74bbf9c2ac2569043871da6aa82b63ac21987a25

Comment (imjoehaines) [reply composer.json]: > Could be - Wasn't sure if you wanted to support V8 as it's not EOL yet

This is just `require-dev` so it only affects the library's test suite 🙂

**Comment by imjoehaines** [comment]
Thanks!

**Comment by imjoehaines** [comment]
Published as v1.0.1

**Pull request merged**